tests/test_stumped.py

```diff
@@ -29,6 +29,8 @@ def dask_client():
     ),
 ]
 
+substitution_values = [np.nan, np.inf]
```

Comment:
I don't think these three lines are needed since `np.nan`/`np.inf` testing is no longer performed in this test file